lib/mpl_toolkits/axes_grid1/axes_size.py

```diff
@@ -19,12 +19,27 @@ class _Base:
     def __rmul__(self, other):
         return Fraction(other, self)
 
+    def __mul__(self, other):
+        return Fraction(self, other)
```

Comment:
Looking at the implementation of `Fraction` this does not work. I'm sorry, this is not well documented, but the Fraction signature is `def __init__(self, fraction : numbers.Real, ref_size : _Base)`, meaning a fraction of the given size. So `0.3 * mysize` is 30% of `mysize`. If anything, you'd have to return `Fraction(other, self)` as well. But since this is basically a scaling factor, it may be better to only accept the factor on the left.